src/OpenTelemetry.Exporter.Stackdriver/StackdriverTraceExporter.cs

```diff
@@ -23,14 +24,14 @@
 using Google.Cloud.Trace.V2;
 using Grpc.Core;
 using OpenTelemetry.Exporter.Stackdriver.Implementation;
-using OpenTelemetry.Trace.Export;
+using OpenTelemetry.Trace;
 
 namespace OpenTelemetry.Exporter.Stackdriver
 {
     /// <summary>
     /// Exports a group of spans to Stackdriver.
     /// </summary>
-    public class StackdriverTraceExporter : SpanExporter
```

Comment:
Should be in separate PR.

Reply:
I renamed because when we updated, the project didn't compile because we don't expose SpanExporter anymore.